python/core/qgsvectorlayer.sip

```diff
@@ -662,11 +662,11 @@ class QgsVectorLayer : QgsMapLayer
      * @param description
      * @param useAsDefault
      * @param uiFileContent
-     * @param msgError
+     * @param msgError (out)
      */
     virtual void saveStyleToDatabase( const QString& name, const QString& description,
                                       bool useAsDefault, const QString& uiFileContent,
-                                      QString &msgError );
+                                      QString &msgError /Out/ );
```

Comment:
I know that it was non-functional. Fact is, *if* someone used it, for him it's conceptually an API break. And API breaks should be documented as good as possible. I guess quite some functions in this document are probably unused. But on the other hand, I'm sure we would all wonder what actually gets used somewhere out there.